@types/ol-ext/interaction/TouchCursorDraw.d.ts

```diff
@@ -13,7 +13,7 @@ export interface Options extends InteractionOptions {
     coordinate?: Coordinate;
     buttons?: Button[];
     maxButtons?: number;
-    type?: GeometryType;
+    type?: typeof GeometryType;
```

Comment:
Ah nice! I think that was one of the things where I struggled when I tried to use the native OL version types!